### DISCUSSION TITLE
X14 ELRS Missing

### DISCUSSION THREAD

**Discussion by MadRC** [FrSkyRC/ETHOS-Feedback-Community#4655] [Q&A]
Hi, on X14 on latest firmware I have no options under the external module for ELRS like I do on the X18. I can select the Twin Module and select ELRS but that does not connect to any of my modules. 

Is there a reason there is no ELRS under the external module selection on X14 ?

**Comment by robthomson** [comment]
Works perfectly on my end.


So you see the elrs menu.

If you select it.. What happens?

Blank screen?

What module you using?

**Reply by MadRC** [reply]
I only see the ELRS option under Type: Twin Lite Pro and then Protocol has ELRS, I don't have ELRS under the Type option at all, is this correct as the T18 has ELRS under the Type on its own ?

All I have is 

- XJT Lite
- R9M Lite
- R9M Lite Access
- R9M Lite Pro Access
- Twin Lite Pro
- PPM
- Sbus


The modules are powering up ok and I have tried 4 modules, Radiomaster Bandit and Ranger Nano as well as BetaFPV ones and all are not connecting, the ones with displays say no handset. 

Under the options I get "No response from Module"

For Reference FW is 1.5.18, EU #5edfa248

**Reply by MadRC** [reply]
![20241023_221439](https://github.com/user-attachments/assets/36e3f483-842e-43d8-a916-1621ae2e8714)
![20241023_221455](https://github.com/user-attachments/assets/b01e0742-976e-41bb-95a8-087ccc57102a)

**Comment by robthomson** [comment]
Looks to me like you have no elrs menu - so likely you have not installed the lua.

https://github.com/FrSkyRC/ETHOS-Feedback-Community/tree/1.5/lua/modules/elrs
